test/e2e_test_cluster_image_policy_with_tsa.sh

```diff
@@ -171,13 +171,13 @@ echo '::endgroup::'
 # Sign it with key
 echo '::group:: Sign demoimage2 with key, and add to rekor and TSA'
 export TSA_URL=`kubectl -n tsa-system get ksvc tsa -ojsonpath='{.status.url}'`
-COSIGN_EXPERIMENTAL=1 COSIGN_PASSWORD="" cosign sign --key cosign.key --allow-insecure-registry --rekor-url ${REKOR_URL} --timestamp-server-url ${TSA_URL} ${demoimage2}
+COSIGN_EXPERIMENTAL=1 COSIGN_YES="true" COSIGN_PASSWORD="" cosign sign --key cosign.key --allow-insecure-registry --rekor-url ${REKOR_URL} --timestamp-server-url ${TSA_URL} ${demoimage2}
```

Comment:
Maybe remove `COSIGN_EXPERIMENTAL` everywhere since it doesn't do anything anymore?

Reply:
Yeah, that will be done in a separate PR.

Reply:
https://github.com/sigstore/policy-controller/pull/535#issuecomment-1407472531